### PR TITLE
Downloading images from S3 bucket and resizing them

### PR DESCRIPTION
**Pull request by kriti0111**
Downloading images from S3 bucket and resizing them using Image crate

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,4 +1,51 @@
...
+use futures::{TryStreamExt};
+use image::imageops::FilterType;
+
+const BUCKET_NAME :&str = "rust-image-resizing";
+const IMAGE_NAME:&str = "rust-image.jpg";
+
 fn main(){
-        println!("hello world");
+        println!("Downloading file from S3 bucket...");
+        let s3 = S3Client::new(Region::ApSoutheast1);
```
Comment (kahlil29) [inline src/main.rs]: Add the region name as a `const` along with the bucket name and image name. 
See [here](https://github.com/Holmusk/rust-foodScoring-service/blob/5f6f0264936e67f9abec1505e956c552b5ba578c/src/rating_utils.rs#L88) for reference on how to parse the region name.

File: src/main.rs [hunk 2]
```diff
@@ -1,4 +1,51 @@
...
+                                Ok(objbytes) => objbytes,
+                                Err(converterr) => panic!("Couldn't convert S3 Object to Image Bytes! {}", converterr),
+                            };
+        let bytes_mutref = s3_object_bytes_mut.as_ref();
+        
+        resize_image(bytes_mutref);
+           
+    }
+
+    fn resize_image(bytes_img : &[u8]){
```
Comment (kahlil29) [inline src/main.rs]: I think this function should return the same type that it takes as input

**Review by kahlil29 — CHANGES_REQUESTED** [review]
Let's move the code into a file named `img_resizing_lambda.rs`. 
Also see if you can get this formatter running for the project: https://github.com/rust-lang/rustfmt

File: src/img_resizing_lambda.rs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+const REGION_NAME: &str = "ApSoutheast1";
+
+fn main() {
+    println!("Downloading file from S3 bucket...");
+    let awsregion = get_region(REGION_NAME.to_string());
+    let s3 = S3Client::new(awsregion);
+    download_img_from_s3(s3, BUCKET_NAME.to_string(), IMAGE_NAME.to_string());
+}
+
+fn get_region(aws_region_name : String) -> Region{
```
Comment (kahlil29) [inline src/img_resizing_lambda.rs:18]: Can you test this function locally please? 
I think the region string input should be `ap-southeast-1`

**Review by kahlil29 — CHANGES_REQUESTED** [review]

File: src/img_resizing_lambda.rs (hunk 3 above)
Comment (kriti0111) [reply src/img_resizing_lambda.rs:18]: Tested this function by printing the region value in Ok as below - 
```
        match aws_region_name.parse::<Region>() {
            Ok(valid_region) => {println!("valid region {:?}",valid_region);
            valid_region
            } ,
            // Default fallback Region (Singapore)
            Err(_) =>  Region::ApSoutheast1,
        }
```
Giving string input(aws_region_name) as `ap-southeast-1' and 'ApSoutheast1' works.

**Review by kahlil29 — APPROVED** [review]

**Pull request merged**